examples/llm/web-frontend/public/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="An app running LLMs on Linera."
+    />
+    <title>Linera Bot</title>
```

Comment:
Or LineraGPT, since you use that elsewhere? :robot: 

```suggestion
    <title>LineraGPT</title>
```